Scripts/BuildAndTest.ps1

```diff
@@ -63,9 +63,12 @@ If (Test-Path "..\bld") {
 if (-not $NoBuild) {
     Write-Information "Building RE2.Native.sln (MSBuild)..."
     MSBuild "$RepoRoot\Src\RE2.Native.sln" /p:Configuration=$Configuration /p:Platform="Any CPU"
+    
+    Write-Information "Restoring SarifPatternMatcher.sln (dotnet)..."
+    dotnet restore $RepoRoot\Src\SarifPatternMatcher.sln
```

Comment:
>dotnet restore $RepoRoot\Src\SarifPatternMatcher.sln [](start = 4, length = 52)

Azure pipeline is having a hard time without this. So, for now i'm forcing the restore.